### PR TITLE
Add Brainboard in tools

### PR DESCRIPTION
**Pull request by Boulet-**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -223,6 +223,7 @@ For more Community Modules not listed here please see the [Terraform Module Regi
 - [astro](https://github.com/uber/astro/) - Astro is a tool for managing multiple Terraform executions as a single command. :ghost:
 - [atlantis](https://github.com/runatlantis/atlantis) - Unified workflow for collaborating on Terraform through GitHub.
 - [blast radius](https://github.com/28mm/blast-radius) - Interactive visualizations of Terraform dependency graphs. :skull:
+- [Brainboard](https://www.brainboard.co) - Visually Design, Deploy & Manage modern cloud infrastructures starting from any Cloud Provider - AWS, GCP, Azure
```
Comment (chenrui333) [inline README.md:226]: This is not OSS tool, right? Then it should be in terraform enterprise?

Comment (chenrui333) [reply README.md:226]: cc @Boulet-

**Comment by chenrui333** [comment]
@Boulet- gonna close now, file free to file a new PR.

**Comment by Boulet-** [comment]
Sorry about that, I'll fill a new PR 👍

**Comment by chenrui333** [comment]
Thank you!

**Pull request closed without merging**